modules/rdg/src/userobjects/SlopeLimitingBase.C

```diff
@@ -52,14 +52,6 @@ SlopeLimitingBase::initialize()
   _lslope.clear();
 }
 
-void
-SlopeLimitingBase::threadJoin(const UserObject & y)
-{
-  const SlopeLimitingBase & pps = static_cast<const SlopeLimitingBase &>(y);
-
-  _lslope.insert(pps._lslope.begin(), pps._lslope.end());
```

Comment:
Yes, it inherits from `GeneralUserObject`, because we need extra sweeps through mesh and that was the best place to do so.

We would like to run threaded inside it, but it was not done yet. There were issues with threading when we did the initial MR. It can be that removing `threadJoin` will be fine.